pipeline/outputs/tcp-and-tls.md

```diff
@@ -28,36 +28,59 @@ The following parameters are available to configure a secure channel connection
 
 ### Command Line
 
+#### JSON format
+
 ```bash
-$ bin/fluent-bit -i cpu -o tcp://127.0.0.1:5170 -p format=json_lines -v
+$ docker run --rm -it --network=host fluent/fluent-bit /fluent-bit/bin/fluent-bit -i cpu -o tcp://127.0.0.1:5170 -p format=json_lines -v
```

Comment:
All the other docs (e.g. https://docs.fluentbit.io/manual/pipeline/outputs/http#command-line) use the command line directly, any particular reason for this change?

Reply:
Mostly it doesn't require building/installing locally. I've been updating the docs as I go along but I'll leave to @lecaros to determine what he prefers.